### PR TITLE
Added MacOS tests for T1115

### PR DESCRIPTION
**Pull request by haresudhan**
**Details:**
Added MacOS tests for T1115 using pbcopy and pbpaste.

**Testing:**
Tested locally in my machine. 

**Associated Issues:**
None

### CODE REVIEW

File: atomics/T1115/T1115.yaml [hunk 1]
```diff
@@ -26,4 +26,13 @@ atomic_tests:
       echo Get-Process | clip
       Get-Clipboard | iex
     name: powershell
+- name: Execute commands from clipboard
+  description: Echo a command to clipboard and execute it
+  supported_platforms:
+  - macos
+  executor:
+    command: |-
+      echo ifconfig | pbcopy
```
Comment (rc-grey) [inline atomics/T1115/T1115.yaml:35]: Did you want the literal string `ifconfig` to be in the clipboard? Or did you want the output of the `ifconfig` command in the clipboard?

If the latter you want `ifconfig | pbcopy`

File: atomics/T1115/T1115.yaml [hunk 2]
```diff
@@ -26,4 +26,13 @@ atomic_tests:
...
+      $(pbpaste)
+    name: sh
```
Comment (rc-grey) [inline atomics/T1115/T1115.yaml:37]: Call out bash here, since the command substitution you're using is bourned based (or zsh)

**Review by rc-grey — CHANGES_REQUESTED** [review]

File: atomics/T1115/T1115.yaml (hunk 1 above)
Comment (haresudhan) [reply atomics/T1115/T1115.yaml:35]: Thanks @rc-grey. I want the literal string ifconfig in the clipboard. Lets say attacker tries to execute commands in the clipboard.

Comment (rc-grey) [reply atomics/T1115/T1115.yaml:35]: Doh - my brain was mush - completely read that wrong yesterday. Looks good

**Review by rc-grey — APPROVED** [review]

**Pull request merged**